deploy/olm-catalog/performance-addon-operator/4.8.0/performance-addon-operator.v4.8.0.clusterserviceversion.yaml

```diff
@@ -239,11 +239,23 @@ spec:
                 image: REPLACE_IMAGE
                 imagePullPolicy: Always
                 name: performance-operator
-                resources: {}
+                resources:
+                  requests:
+                    cpu: 10m
```

Comment:
is this realistic or a placeholder value we set to be able to have `requests`?
Same for memory below

Reply:
in general, I just took some values from https://github.com/openshift/machine-config-operator/blob/master/manifests/machineconfigcontroller/deployment.yaml and halved it, because we do much less work than MC controller.

But mostly I added it to be sure that our pod under the buarstable QoS and not under the best-effort one.

Reply:
ok, makes sense. And since these are requests (not limits), it seems like a good start.